status-im/status-react
# Add device name to pairing

Add device name to pairing section.

The first time a user access the devices section, they will be prompted to set a name for the device.

Once a name is set, it will be exposed to other devices when Pair this device is enabled (not through automatic discovery).

Also moves Devices section outside of advanced.
![2019-01-14-154115_3840x1080_scrot](https://user-images.githubusercontent.com/1017008/51119784-068fa300-1814-11e9-8c29-167af2561261.png)


### Testing

1) Recover account A1 & A2
2) Enable development mode on both.
3) Go to settings -> Devices. You will be asked to set a name
4) Once a name is set, click on Pair this device (on both A1 & A2)
5) You should see A1 and A2 names in the list

In case we don't have this info (automatically discovered), "No Info (device-id)" will be shown. To see this you can recover A1 & A2, enable development mode, but don't pair. Send a 1-to-1 message to anyone from A1, A2 should now have A1 in the list.

Attached a screenshot @errorists what do you think?

status: ready
